VMEncryption/main/oscrypto/ubuntu_2004/Ubuntu2004EncryptionStateMachine.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+#
+# Copyright (C) Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import inspect
+import os
+import sys
+import traceback
+from time import sleep
+
+scriptdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+maindir = os.path.abspath(os.path.join(scriptdir, '../../'))
+sys.path.append(maindir)
+sixdir = os.path.abspath(os.path.join(scriptdir, '../../six'))
+sys.path.append(sixdir)
+transitionsdir = os.path.abspath(os.path.join(scriptdir, '../../transitions'))
+sys.path.append(transitionsdir)
+
+from oscrypto import *
```

Comment:
Can we avoid * imports here?

Reply:
Good catch

Reply:
Would like to fix these in all state machines, not just Ubuntu 20.04.  Created an issue to track separately https://github.com/Azure/azure-linux-extensions/issues/1322 and clean up all at once in a separate change.